hexrd/ui/load_panel.py

```diff
@@ -147,8 +147,9 @@ def config_changed(self):
         self.ui.file_options.setRowCount(0)
         self.reset_data()
         self.enable_read()
-        HexrdConfig().load_panel_state = {}
-        self.state = self.setup_processing_options()
+        HexrdConfig().clear_images()
+        self.setup_gui()
+        self.parent().image_tab_widget.load_images()
```

Comment:
I *think* that the main window will already be updating the images [here](https://github.com/HEXRD/hexrdgui/blob/378bc1fe62feb8220f2360e2bd673c513b30a280/hexrd/ui/main_window.py#L171) (which should get called after this block). So this might not be needed. I'm going to be making sure the dummy images get loaded on new config in #260 as well.

Reply:
Removed.

Reply:
Thanks Brianna!